### PR TITLE
HW-5 Add library to project

### PR DESCRIPTION
**Pull request by HannaKishkinova**
* Added checkbox lib

### CODE REVIEW

File: libs/checkbox/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './lib/checkbox.module';
```
Comment (gund) [inline libs/checkbox/src/index.ts:1]: You should also export checkbox component to be part of the public API.

File: libs/checkbox/src/lib/checkbox/checkbox.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+})
+export class CheckboxComponent implements OnInit {
+  @Input() isDisabled = false;
+  @Input() isChecked = false;
+  @Output() checkedChange = new EventEmitter<boolean>();
+
+  ngOnInit() {}
+
+  toggleCheck() {
+    this.checkedChange.emit();
```
Comment (gund) [inline libs/checkbox/src/lib/checkbox/checkbox.component.ts:16]: You forgot to pass the value of the checkbox to emitted event here.

File: libs/checkbox/src/lib/checkbox/checkbox.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+
+@Component({
+  selector: 'ng-workspace05-nx-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.css']
+})
+export class CheckboxComponent implements OnInit {
+  @Input() isDisabled = false;
+  @Input() isChecked = false;
```
Comment (gund) [inline libs/checkbox/src/lib/checkbox/checkbox.component.ts:10]: Inputs should've been named without `is` prefix =)

File: libs/search/src/lib/search/search.component.html [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
-
+<br>
 <ng-workspace05-nx-input></ng-workspace05-nx-input>
-<label>
-  <input type="checkbox" [(ngModel)]="isGlobalSearch" /> Global search?
-</label>
+<br>
+<ng-workspace05-nx-checkbox
+  [isDisabled]="false"
+  [isChecked]="isGlobalSearch"
```
Comment (gund) [inline libs/search/src/lib/search/search.component.html:7]: You can use two-way binding here to get rid from output and manual property update.

For more info check https://gist.github.com/gund/3ae21110e7fd9cf63cb72c54e4d6a4ac

File: libs/checkbox/src/lib/checkbox/checkbox.component.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<div>
+  <label>
+    <input
+      type="checkbox"
+      [disabled]="isDisabled"
+      [checked]="isChecked"
```
Comment (gund) [inline libs/checkbox/src/lib/checkbox/checkbox.component.html:6]: You should use two-way binding to automatically have updated value in your property to then pass it to the output.

**Review by gund — COMMENTED** [review]
Good job. There are some improvements that can be made to simplify checkbox component usage.

File: libs/checkbox/src/index.ts (hunk 1 above)
Comment (HannaKishkinova) [reply libs/checkbox/src/index.ts:1]: done

File: libs/checkbox/src/lib/checkbox/checkbox.component.ts (hunk 2 above)
Comment (HannaKishkinova) [reply libs/checkbox/src/lib/checkbox/checkbox.component.ts:16]: done

File: libs/checkbox/src/lib/checkbox/checkbox.component.ts (hunk 3 above)
Comment (HannaKishkinova) [reply libs/checkbox/src/lib/checkbox/checkbox.component.ts:10]: done

**Pull request closed without merging**